emcc.py

```diff
@@ -2276,23 +2274,20 @@ def phase_linker_setup(options, state, newargs):
   if settings.MIN_CHROME_VERSION <= 37:
     settings.WORKAROUND_OLD_WEBGL_UNIFORM_UPLOAD_IGNORED_OFFSET_BUG = 1
 
-  # 10.19.0 is the oldest version of node that we do any testing with.
+  # 16.20.0 is the oldest version of node that we do any testing with.
   # Keep this in sync with the test-node-compat in .circleci/config.yml
   # and MINIMUM_NODE_VERSION in tools/shared.py
   if settings.MIN_NODE_VERSION:
-    if settings.MIN_NODE_VERSION < 101900:
-      exit_with_error('targeting node older than 10.19.00 is not supported')
-    if settings.MIN_NODE_VERSION >= 150000:
-      default_setting('NODEJS_CATCH_REJECTION', 0)
+    if settings.MIN_NODE_VERSION < 162000:
+      exit_with_error('targeting node older than v16.2 is not supported')
```

Comment:
Wait, I'm confused - `MIN_NODE_VERSION` is for the generated code, which contradicts the changelog entry.

Are both being updated in this PR?

If we want to change the generated code version, can we not support older node by using transpilation?

Reply:
Sorry you are correct, I also bumped min runtime version we support from 150000 to 162000.   This keeps the testing matrix a little simple since it avoids us having to do specific testing with v15.  But maybe its worth keeping this change focused on the build-time version.

Reply:
I decided to stick with the update of both version.    The min output version is much more minor bump